CMakeLists.txt

```diff
@@ -1,9 +1,7 @@
 cmake_minimum_required(VERSION 3.13)
-
-project(passgen LANGUAGES CXX)
-
+project(passgen VERSION 1.0.0 LANGUAGES CXX)
```

Comment:
A more Ideal way is to set CMake variables to store MAJOR, MINOR and REVISION numbers. Having version in top level CMakeLists.txt is ideal.
Example:-

```
set(passgen_VERSION MAJOR 1)
set(passgen_VERSION_MINOR 0)
set(passgen_VERSION_REVISION 0)

```
No need for VERSION in `project` command here